docs/reference.rst

```diff
@@ -709,6 +709,22 @@ Faker
                 date_end=datetime.date(2020, 5, 31),
             )
 
+    Since Faker 4.9.0 version (see `Faker documentation <https://faker.readthedocs.io/en/latest/>`_),
```

Comment:
That link does not show the feature being introduced in Faker. https://github.com/joke2k/faker/releases/tag/v4.9.0 would be more accurate.